client/state/wp-api/posts/fetch-post.js

```diff
@@ -0,0 +1,15 @@
+import wpcom from 'lib/wp';
+
+import { receivePost } from 'state/posts/actions';
+
+const savePost = dispatch => post => dispatch( receivePost( post ) );
+const failPost = error => console.log( error );
+
+export const fetchPost = ( { dispatch }, { siteId, postId } ) => {
```

Comment:
agreed. part of this is me trying to keep things clean for the PR. I started trying to simply replace the work that the post query component does but there's too much ancillary logic there and too many name clashes already